src/components/nav-drawer/nav-drawer.ts

```diff
@@ -80,6 +80,7 @@ export default class IgcNavDrawerComponent extends LitElement {
     };
   }
 
+  /* eslint-disable lit-a11y/click-events-have-key-events */
```

Comment:
Might be a future addition to handle ESC to close the drawer for example. https://m3.material.io/components/navigation-drawer/accessibility is still pending, but I've seen some apps close with KB nav. However, there's a good chance this is left as an app scenario as we prob can't manage the focus state after close correctly, in which case this could be fine like that.